components/bl-stopwatch/src/index.js

```diff
@@ -27,34 +27,24 @@ const timeFormatter = {
 const initTime = { seconds: 0, minutes: 0, hours: 0 };
 
 export default function Stopwatch({ component }) {
-  const { display, classList, timeFormat, tickRate } = component;
+  const { display, classList, timeFormat, decimalPlaces } = component;
 
   const [time, setTime] = useObjectState(initTime);
 
   const timerRef = useRef();
 
-  const validTickRate = useMemo(() => {
-    if (tickRate < 0) {
-      console.warn('Stopwatch Scale can\'t be less than 0');
-
-      return 0;
-    }
-
-    return tickRate;
-  }, []);
-
   component.start = () => {
     if (!timerRef.current) {
       const startTime = Date.now();
-      const interval = validTickRate === 0 ? 1000 : 1;
+      const interval = Number(decimalPlaces) === 0 ? 1000 : 1;
```

Comment:
Still not clear for me why we run this interval every 1ms if precision is 10ms or 100ms.

Is there are any reason for it?